app/code/Magento/Multishipping/Test/Mftf/Test/StoreFrontCheckVatIdAtAccountCreateWithMultishipmentTest.xml

```diff
@@ -0,0 +1,55 @@
+    <test name="StoreFrontCheckVatIdAtAccountCreateWithMultishipmentTest">
+        <annotations>
+            <features value="Multishipment"/>
+            <stories value="Checking vat id field at account create page  with 'Check Out with Multiple Addresses'"/>
+            <title value="Checking vat id field at account create page  with 'Check Out with Multiple Addresses'"/>
+            <description value="'VAT Number' field should be available at create account page  if 'Show VAT Number on Storefront' is Yes"/>
+            <severity value="MAJOR"/>
+            <testCaseId value="MC-40016"/>
+            <group value="Multishipment"/>
+        </annotations>
+        <before>
+            <magentoCLI command="config:set customer/create_account/vat_frontend_visibility 1" stepKey="showVatNumberOnStorefrontYes"/>
+            <actionGroup ref="CliCacheFlushActionGroup" stepKey="flushCacheBefore">
+                <argument name="tags" value="config"/>
+            </actionGroup>
+            <createData stepKey="category" entity="SimpleSubCategory"/>
+            <createData stepKey="product1" entity="SimpleProduct">
+                <requiredEntity createDataKey="category"/>
+            </createData>
+        </before>
+        <!-- Add product to the cart -->
+        <amOnPage url="$$product1.name$$.html" stepKey="goToProductPage"/>
+        <actionGroup ref="AddToCartFromStorefrontProductPageActionGroup" stepKey="addProductToCart">
+            <argument name="productName" value="$$product1.name$$"/>
+        </actionGroup>
+        <!-- Check Out with Multiple Addresses -->
+        <actionGroup ref="StorefrontOpenCartFromMinicartActionGroup" stepKey="openCart"/>
+        <waitForElementVisible selector="{{MultishippingSection.shippingMultipleCheckout}}" stepKey="waitMultipleAddressShippingButton"/>
+        <click selector="{{MultishippingSection.shippingMultipleCheckout}}" stepKey="clickToMultipleAddressShippingButton"/>
+        <!--Create an account-->
+        <waitForElementVisible selector="{{StorefrontCustomerSignInPopupFormSection.createAnAccount}}" stepKey="waitCreateAnAccountButton"/>
+        <click selector="{{StorefrontCustomerSignInPopupFormSection.createAnAccount}}" stepKey="clickOnCreateAnAccountButton"/>
+        <waitForPageLoad stepKey="waitForCreateAccountPageToLoad"/>
+        <!--Check the VAT Number field-->
+        <seeElement selector="{{StorefrontCustomerAddressSection.vatId}}" stepKey="assertVatIdField"/>
+        <after>
+            <magentoCLI command="config:set customer/create_account/vat_frontend_visibility 0" stepKey="showVatNumberOnStorefrontNo"/>
+            <actionGroup ref="CliCacheFlushActionGroup" stepKey="flushCacheAfter">
+                <argument name="tags" value="config"/>
+            </actionGroup>
+            <deleteData stepKey="deleteCategory" createDataKey="category"/>
+            <deleteData stepKey="deleteProduct1" createDataKey="product1"/>
+        </after>
```

Comment:
Let's keep this section after the `<before>` node, to follow the same pattern as other tests and the docs https://devdocs.magento.com/mftf/docs/test.html

Reply:
Has been moved according to the pattern.